CapMan.Raylib/Program.cs

```diff
@@ -33,14 +33,15 @@
 game.Player.Y = 23; // * BoardRenderer.CellSize;
 game.Player.X = 14; // * BoardRenderer.CellSize;
 BoardRenderer boardRenderer = new();
+ScoreRenderer scoreRenderer = new();
 
 int boardWidth = game.Board.Columns * BoardRenderer.CellSize;
-int boardHeight = game.Board.Rows * BoardRenderer.CellSize;
+int boardHeight = (game.Board.Rows * BoardRenderer.CellSize) + ScoreRenderer.BlockedHeight;
```

Comment:
What do you think about rendering it to the topside of the board? Similar to the original game: 
![200518114838-05-pac-man-40](https://github.com/CaptainCoderOrg/CapMan/assets/3529380/71628ac0-0676-4efb-9904-4194ade03338)

Reply:
I tried topsize first but I ran into some problems with shifting the board down. Probably because a bunch of code just assumes top/left 0,0 origin. 

For example, I moved the board down by an offset, but CapMan didn't shift and was eating pellets it wasn't over top of.